### PR TITLE
fix: fix broken FTL examples

### PR DESCRIPTION
**Pull request by wesbillman**

### CODE REVIEW

File: examples/go/echo/echo.go [hunk 1]
```diff
@@ -24,12 +24,11 @@ type EchoResponse struct {
...
 //ftl:verb
-//ftl:ingress GET /echo
 func Echo(ctx context.Context, req EchoRequest) (EchoResponse, error) {
 	fmt.Println("Echo received a request!")
 	tresp, err := sdk.Call(ctx, time.Time, time.TimeRequest{})
 	if err != nil {
 		return EchoResponse{}, err
 	}
-	return EchoResponse{Message: fmt.Sprintf("Hello, %s!!! It is %s!", req.Name.Default("anonymous"), tresp.Time)}, nil
+	return EchoResponse{Message: fmt.Sprintf("Hello, %s!!! It is %s!", req.Name.Default("anonymous"), tresp)}, nil
```
Comment (alecthomas) [inline examples/go/echo/echo.go:33]: Is this correct?

File: examples/kotlin/ftl-module-ad/src/main/kotlin/ftl/ad/Ad.kt [hunk 2]
```diff
@@ -16,12 +19,14 @@ class AdModule {
   private val database: Map<String, Ad> = loadDatabase()
 
   @Verb
-  @Ingress(Method.GET, "/get")
-  fun get(context: Context, req: AdRequest): AdResponse {
-    return when {
-      req.contextKeys != null -> AdResponse(ads = contextualAds(req.contextKeys))
-      else -> AdResponse(ads = randomAds())
+  @Ingress(Method.GET, "/get", HTTP)
```
Comment (alecthomas) [inline examples/kotlin/ftl-module-ad/src/main/kotlin/ftl/ad/Ad.kt:22]: Weird, we should get rid of `HTTP` and call this `@HttpIngress`. It's not going to be usable for any other type such as gRPC anyway.

**Review by alecthomas — APPROVED** [review]

File: examples/kotlin/ftl-module-ad/src/main/kotlin/ftl/ad/Ad.kt (hunk 2 above)
Comment (wesbillman) [reply examples/kotlin/ftl-module-ad/src/main/kotlin/ftl/ad/Ad.kt:22]: Yeah good call. I'll make a ticket to clean this up.
https://github.com/TBD54566975/ftl/issues/874

File: go-runtime/compile/build.go [hunk 3]
```diff
@@ -145,6 +145,16 @@ var scaffoldFuncs = scaffolder.FuncMap{
...
+				for _, tp := range n.TypeParameters {
+					tpRef, err := schema.ParseDataRef(tp.String())
+					if err != nil {
+						return err
+					}
+					if tpRef.Module != "" && tpRef.Module != m.Name {
+						imports[path.Join("ftl", tpRef.Module)] = "ftl" + tpRef.Module
+					}
+				}
+
```
Comment (wesbillman) [inline go-runtime/compile/build.go:157]: @alecthomas I uncovered this with the tests. It seemed like imports for external modules in typeparameters weren't being imported. Does this fix look ok to you?

Comment (alecthomas) [reply go-runtime/compile/build.go:157]: Ah that's quite possible. Looks about right.

**Pull request merged**